libs/ts/base-utils/src/evm/networks.ts

```diff
@@ -252,716 +252,1203 @@ export const networkMetadata = {
+    explorer: {
+      type: 'blockscout',
+      webUrl: 'https://explorer.testnet.aurora.dev',
+      apiUrl: 'https://explorer.testnet.aurora.dev/api',
+    },
     currency: Currency.ETH,
   },
   'avalanche-mainnet': {
     chainId: 43114,
     isTestnet: false,
-    explorerUrl: 'https://snowtrace.io',
+    explorer: {
+      type: 'routescan',
+      webUrl: 'https://snowtrace.io',
+      apiUrl: 'https://api.routescan.io/v2/network/mainnet/evm/43114/etherscan',
+    },
     currency: Currency.AVAX,
   },
   'avalanche-fuji': {
     chainId: 43113,
     isTestnet: true,
-    explorerUrl: 'https://testnet.snowtrace.io',
+    explorer: {
+      type: 'routescan',
+      webUrl: 'https://subnets-test.avax.network/c-chain',
+      apiUrl: 'https://api.routescan.io/v2/network/mainnet/evm/43113/etherscan',
+    },
     currency: Currency.AVAX,
   },
   'base-mainnet': {
     chainId: 8453,
     isTestnet: false,
-    explorerUrl: 'https://basescan.org',
+    explorer: {
+      type: 'blockscout',
+      webUrl: 'https://base.blockscout.com',
+      apiUrl: 'https://base.blockscout.com/api',
+    },
     currency: Currency.ETH,
   },
   'base-sepolia': {
     chainId: 84532,
     isTestnet: true,
-    explorerUrl: 'https://sepolia.basescan.org',
+    explorer: {
+      type: 'blockscout',
+      webUrl: 'https://base-sepolia.blockscout.com',
+      apiUrl: 'https://base-sepolia.blockscout.com/api',
+    },
     currency: Currency.ETH,
   },
   basecamp: {
     chainId: 123420001114,
     isTestnet: true,
-    explorerUrl: 'https://basecamp.cloud.blockscout.com',
+    explorer: {
+      type: 'blockscout',
+      webUrl: 'https://basecamp.cloud.blockscout.com',
+      apiUrl: 'https://basecamp.cloud.blockscout.com/api',
+    },
     currency: Currency.CAMP,
   },
   'berachain-mainnet': {
     chainId: 80094,
     isTestnet: false,
-    explorerUrl: 'https://berascan.com',
+    explorer: {
+      type: 'etherscan',
+      webUrl: 'https://berascan.com',
+      apiUrl: 'https://berascan.com/api',
+    },
     currency: Currency.BERA,
   },
   'berachain-bartio': {
     chainId: 80084,
     isTestnet: true,
-    explorerUrl: 'https://bartio.beratrail.io',
+    explorer: {
+      type: 'routescan',
+      webUrl: 'https://bartio.beratrail.io',
+      apiUrl: 'https://api.routescan.io/v2/network/testnet/evm/80084/etherscan',
+    },
     currency: Currency.BERA,
   },
   'berachain-bepolia': {
     chainId: 80069,
     isTestnet: true,
-    explorerUrl: 'https://bepolia.beratrail.io',
+    explorer: {
+      type: 'routescan',
+      webUrl: 'https://bepolia.beratrail.io',
+      apiUrl: 'https://api.routescan.io/v2/network/testnet/evm/80069/etherscan',
+    },
     currency: Currency.BERA,
   },
   'bitlayer-mainnet': {
     chainId: 200901,
     isTestnet: false,
-    explorerUrl: 'https://www.btrscan.com',
+    explorer: {
+      type: 'bitlayer',
+      webUrl: 'https://www.btrscan.com',
+      apiUrl: null,
```

Comment:
```suggestion
      apiUrl: 'https://api.btrscan.com/scan/api',
```

Source: https://docs.bitlayer.org/docs/Build/BitlayerScanAPIs/Accounts/